catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/associations/Associated.java

```diff
@@ -264,20 +276,22 @@ private Filter forChildAssociations(Metacard metacard) {
 
     }
 
-    private Map<String, Metacard> query(Filter filter)
+    private Map<String, Metacard> query(List<String> writableSourceIds, Filter filter)
             throws UnsupportedQueryException, SourceUnavailableException, FederationException {
         if (filter == null) {
             return Collections.emptyMap();
         }
 
-        QueryResponse query = catalogFramework.query(new QueryRequestImpl(new QueryImpl(filter,
+        QueryRequestImpl queryRequest = new QueryRequestImpl(new QueryImpl(filter,
                 1,
-                0,
+                EndpointUtil.DEFAULT_QUERY_PAGE_SIZE,
```

Comment:
This query is called to get the associated Metacards by a Metacard ID.  I'm not sure the amount of associated Metacards would ever get to this point.  To avoid blowing up SolrCloud, it may be more reasonable to reduce this  page size to a few thousand records.